config/config.go

```diff
@@ -50,6 +50,7 @@ type Config struct {
 	DefaultHTTPStatus               int            `yaml:"default-http-status"`
 	PluginDir                       string         `yaml:"plugindir"`
 	LoadBalancerHealthCheckInterval time.Duration  `yaml:"lb-healthcheck-interval"`
+	LoadBalancerAlgorithm           string         `yaml:"lb-algorithm"`
```

Comment:
Should be something like `KubernetesDefaultLoadBalancerAlgorithm`, because it makes only sense there.